### DISCUSSION TITLE
Ability to export workspace without cookies

### DISCUSSION THREAD

**Discussion by fabiojaniolima** [Kong/insomnia#3522] [Ideas]
**Is your feature request related to a problem? Please describe.**
Considering the privacy factor I would say that it is a problem related to each context of use.

After exporting the workspace and opening the JSON file, it creates content that presents a cookie with the data definitions of my private workspace, that is, the export does not cover this privacy feature.

Example:
```
{
    "_id":"jar_sdkjsdf823492873423423424",
    "cookies":[
        {
            "creation":"2020-05-08T00:15:27.005Z",
            "domain":"DOMAIN.OMITTED",
            "hostOnly":true,
            "id":"078496558888436",
            "key":"BIGipServerPOOL_HOST-OMITTED_PORT-OMITTED",
            "lastAccessed":"2020-05-08T00:15:27.005Z",
            "path":"/",
            "value":"1310013962.61475.0000"
        }
    ],
    "created":1588854419453,
    "modified":1588896927006,
    "name":"Default Jar",
    "parentId":"wrk_af7d97f6c23c4ec9a07d16ac1d78382",
    "_type":"cookie_jar"
}
```

> * .OMITTED: represents private data that was exported in the context of the Cookie.

**Describe the solution you'd like**
A clear and concise description of what you want to happen. Remove Cookies when performing an export.

**Comment by nijikokun** [comment]
Hello, yes, we full export all data so you are able to transport it to other applications and leave the burden of ensuring that whatever you are putting the export into you trust and have confirmed it to be a secure or secure enough environment.

These exports are often times used for replication purposes.

I could see maybe an option to export with certain things removed.

---

Adding the needs discussion label and revising title to reflect ask.

**Comment by shravan2x** [comment]
Are there any updates here? Insomnia seems to silently export cookies without even a warning.

**Comment by sqllyw** [comment]
really need this feature, any updates?
